docs/react-testing-library/example-intro.mdx

```diff
@@ -4,6 +4,65 @@ title: Example
 sidebar_label: Example
 ---
 
+## Quickstart
+
+This is a very simple setup to get you started. 
```

Comment:
Let's drop the "simple". This may apply to us but not necessarily for learners.

Reply:
@eps1lon Done!